templates/base/mock-api/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "json-mock-api",
+  "version": "1.0.0",
+  "description": "A simple JSON mock API",
+  "main": "index.js",
+  "scripts": {
+    "start": "json-server --watch src/db.json",
```

Comment:
@madan180 think of these commends being merged with rest of the project configurations. In that way it needs to be modified. 
Can you please rename these scripts to following: 

start -->  mock:rest
start:gql -->  mock:graphql

But the way you have set it up, it also makes sense to set it up as an optional feature. 

@hiteshparashar4  Can help you setup this as optional feature.

Reply:
Done